src/components/backupWallet/__tests__/ExportWalletData.js

```diff
@@ -6,6 +6,8 @@ import renderer from 'react-test-renderer'
 // utils
 import { getWebRouterComponentWithMocks } from './__util__'
 
+jest.doMock('../../../lib/wallet/GoodWallet')
```

Comment:
please, see https://github.com/GoodDollar/GoodDAPP/pull/2090#discussion_r448216947

Reply:
1. let's mock actually what we need
2. let's mock  on setup and unmock on teardown  (you haven't unmocked the module)